src/metax_api/models/catalog_record.py

```diff
@@ -996,6 +996,13 @@ def catalog_is_pas(self):
     def has_alternate_records(self):
         return bool(self.alternate_record_set)
 
+    def state_is_draft(self, request):
+        from metax_api.services import CommonService
+        if request.query_params.get('draft', None) is not None:
+            return bool(CommonService.get_boolean_query_param(self.request, 'draft') and settings.DRAFT_ENABLED)
+        else:
+            pass
+
```

Comment:
Just this line (along with the from...import) should be enough for this method's contents:

``return CommonService.get_boolean_query_param(self.request, 'draft') and settings.DRAFT_ENABLED``

Reply:
Ok!